# Comparison of imaging based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed paraffin-embedded tumor samples 

Code repository for our paper. Raw data are in the GEO database. Source data of images are in supplementary data in the publication.

The CosMx SMI, MERFISH and Xenium raw data generated in this study have been deposited in the Gene Expression Ominibus (GEO) database (https://www.ncbi.nlm.nih.gov/geo/) under accession code GSE299786, GSE299886 and GSE300007, respectively. 

# Abstract
Imaging-based spatial transcriptomics (ST) is evolving as a pivotal technology in studying tumor biology and associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. We use serial 5 ÔÅ≠m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma samples in tissue microarrays to compare the performance of the ST platforms (CosMx, MERFISH, and Xenium (uni/multi-modal)) in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx, and hematoxylin and eosin staining data. In addition to an objective assessment of automatic cell segmentation and phenotyping, we perform a manual phenotyping evaluation to assess pathologically meaningful comparisons between ST platforms. Here, we show the intricate differences between the ST platforms, reveal the importance of parameters such probe design in determining the data quality, and suggest reliable workflows for accurate spatial profiling and molecular discovery.

